tests/OpenCloud/Tests/ObjectStore/Resource/DataObjectTest.php

```diff
@@ -101,4 +101,21 @@ public function test_Public_Urls()
         $this->assertNotNull($object->getPublicUrl(UrlType::STREAMING));
         $this->assertNotNull($object->getPublicUrl(UrlType::IOS_STREAMING));
     }
+    
+    public function test_Symlink()
+    {
+        $object = $this->container->dataObject('foobar');
+        $this->assertInstanceOf(
+            'Guzzle\Http\Message\Response',
+            $object->symlink('/new_container/new_object')
+        );
```

Comment:
Could you please add another assertion to this test that checks the value of `getManifest` after the test creates the symlink? This will ensure test coverage on the `getManifest` method.